crates/wash-cli/docs/DEVELOPMENT.md

```diff
@@ -56,6 +56,14 @@ To build continuously (thanks to [`cargo-watch`](https://crates.io/crates/cargo-
 ```console
 make build-watch
 ```
+## Testing Local Changes
+
+After making changes to code in `crates/wash-cli`, you can build and run a new version of `wash` similarly to any other Rust binary project with `cargo run`:
+
+```console
+cargo run -- <args>
+```
```

Comment:
```suggestion
```console
cd crates/wash-cli
cargo run -- <args>
```
```

What do you think about being really explicit here that you should run this in the cli directory? Just to avoid devs making changes and then running wasmCloud on accident

Reply:
I agree on adding it, partly because I've made the same mistake of running it in root of wasmcloud and compiling everything.